test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnVerbs.java

```diff
@@ -342,6 +342,12 @@ public static HapiContractCall contractCall(
         return new HapiContractCall(abi, contract, params);
     }
 
+    public static HapiContractCall contractCall(
+            String contract, String functionName, String param) {
+        final var abi = getABIFor(FUNCTION, functionName, contract);
+        return new HapiContractCall(abi, contract, param);
+    }
+
```

Comment:
Since `Utils.asHexedAddress` returns `String`, this overloaded method becomes obsolete. It can be removed.